indico/web/client/js/react/util/html.js

```diff
@@ -26,10 +26,18 @@ export function toClasses(...params) {
  * It takes a function which will be called with `resolve, reject` and render
  * the actual component.
  */
-export function injectModal(renderFunc) {
+export function injectModal(renderFunc, position) {
   const container = document.createElement('div');
   document.body.appendChild(container);
 
+  if (position) {
+    const {left, top} = position;
+    container.style.position = 'absolute';
+    container.style.left = `${left}px`;
+    container.style.top = `${top}px`;
+    container.style.zIndex = '1001';
```

Comment:
not the biggest fan of hardcoding this hear, but i don't have a good idea on how to do it more nicely...

Reply:
Yep, I didn't like it either, but didn't manage to come up with something cleaner 😞